### DISCUSSION TITLE
how to run python app devloped using flet on mobile

### DISCUSSION THREAD

**Discussion by chaitananjfwfuiwbufbrui** [flet-dev/flet#634] [General]
is there any command for build file?
how is it possible to run on mobile?

**Comment by ndonkoHenri** [comment]
- What do you mean by 'build file'?

- It is not actually possible to do mobile apps with Flet. But it's on it's way. Stay tuned. Nevertheless, apps deployed to the web could be installed as PWAs(progressive web app). See "easiest way" here: https://github.com/ndonkoHenri/Flet-Color-Browser#how-to-get-started
